src/EditorFeatures/TestUtilities/EmbeddedLanguages/EmbeddedLanguagesTestConstants.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Security;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities.EmbeddedLanguages
+{
+    internal static class EmbeddedLanguagesTestConstants
```

Comment:
constants to allow us to try out the StringSyntaxAttribute in test code since it's not in an sdk yet our tests are referencing.